spec/lib/dor/was_crawl/warc_extractor_service_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'spec_helper'
+
+RSpec.describe Dor::WasCrawl::WarcExtractorService do
+  describe '#extract' do
+    before do
+      FileUtils.cp_r 'spec/lib/dor/was_crawl/fixtures/workspace/dd111dd1111', 'tmp/'
+    end
+
+    after do
+      FileUtils.rm_rf 'tmp/dd111dd1111'
+    end
+
+    it 'extracts the WARC files' do
+      described_class.extract('tmp/dd111dd1111', 'WACZ-Test.wacz')
+      expect(File.exist?('tmp/dd111dd1111/WACZ-Test-data.warc.gz')).to be true
+      expect(File.exist?('tmp/dd111dd1111/WACZ-Test.wacz')).to be false
```

Comment:
Based on conversation with @justinlittman we decided that since this unique filename issue will be a problem for one-time accessioning of WARCs, and we have an issue to audit the uniqueness of the Path Index #477, that it might be good to revisit our decision to use the Path Index at all and that we might be better off creating the CDXJ index with the relative path in it, rather than just the filename. So I'm going to merge this as is, and create a ticket for that.